js-exercises/last-index-of/lastIndexOf.js

```diff
@@ -0,0 +1,13 @@
+function lastIndexOf(...args) {
+  let searchedValue = args[0];
+  let list = args[1];
+  for (let i = list.length; i >= 0; i--) {
```

Comment:
forEach can be used to iterate over the array. And as the second argument to the callback function to forEach can be used to get the index.